messages/en.json

```diff
@@ -1263,38 +1263,47 @@
   "sharedComponents.GpsPill.searching": {
     "message": "Searching…"
   },
-  "sharedComponents.ProjectInviteBottomSheet.InviteCancelled.canceled": {
-    "message": "Invite Canceled"
+  "sharedComponents.ProjectInviteBottomSheet.InviteBottomSheetContent.InviteAccepted.goToMap": {
+    "message": "Go To Map"
   },
-  "sharedComponents.ProjectInviteBottomSheet.InviteCancelled.close": {
-    "message": "Close"
+  "sharedComponents.ProjectInviteBottomSheet.InviteBottomSheetContent.InviteAccepted.goToSync": {
+    "message": "Go To Sync"
   },
-  "sharedComponents.ProjectInviteBottomSheet.InviteCancelled.projectInviteCanceled": {
-    "message": "Your invitation to {projectName} has been canceled."
+  "sharedComponents.ProjectInviteBottomSheet.InviteBottomSheetContent.InviteAccepted.success": {
+    "message": "Success"
   },
-  "sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.acceptInvite": {
-    "message": "Accept Invite"
+  "sharedComponents.ProjectInviteBottomSheet.InviteBottomSheetContent.InviteAccepted.youHaveJoined": {
+    "message": "You have joined <bold>{projectName}</bold>"
   },
-  "sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.declineInvite": {
-    "message": "Decline Invite"
+  "sharedComponents.ProjectInviteBottomSheet.InviteBottomSheetContent.InviteCanceled.canceled": {
+    "message": "Invite Canceled!"
   },
-  "sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.invitedToJoin": {
-    "message": "You've been invited to join {projectName}"
+  "sharedComponents.ProjectInviteBottomSheet.InviteBottomSheetContent.InviteCanceled.close": {
+    "message": "Close"
   },
-  "sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.joinProject": {
-    "message": "Join Project {projectName}"
```

Comment:
Based on the designs in Figma, I was under the impression that "Project" was part of the actual project name, not something that we always prefix the actual project name with. Could be mistaken though